src/coap_io.c

```diff
@@ -1167,7 +1217,8 @@ coap_write(coap_context_t *ctx,
             if (s->proto == COAP_PROTO_DTLS && s->tls) {
               coap_tick_t tls_timeout = coap_dtls_get_timeout(s);
               while (tls_timeout > 0 && tls_timeout <= now) {
-                coap_log(LOG_DEBUG, "**  %s: DTLS retransmit timeout\n", coap_session_str(s));
+                coap_log(LOG_DEBUG, "**  %s: DTLS retransmit timeout\n",
```

Comment:
While here, the extra LWS has been there before and could have been deleted when touching this line anyway.